app/Models/UserModel.php

```diff
@@ -0,0 +1,149 @@
+<?php
+
+namespace Application\Models;
+
+use Application\Controllers\Controller;
+
+class UserModel extends Controller
```

Comment:
```suggestion
class User
```
Pas la peine de extend de controller il devrait extend in class Model 
et le nom du file aussi

Reply:
I did that 
<img width="411" alt="Capture d’écran 2022-08-19 à 10 13 10" src="https://user-images.githubusercontent.com/18627668/185575089-28dd3c5d-2334-482f-8ee3-60b0dd9f8763.png">

Reply:
it's work quite well